roles/ceph-mon/tasks/set_rgw_pool_pg_num.yml

```diff
@@ -0,0 +1,9 @@
+# Uncomment below lines to create pools for the rgw 
+# TODO: recommend tuning pg_nums for different configurations
+#rgw_pool:
+#  rgw_pool_name1:
```

Comment:
@cbodley are there any default values we can add to this, so the entire file isn't commented out?